web/urls.py

```diff
@@ -135,6 +135,7 @@
     ),
     path("tasks", teacher_view.task_list, name="tasks"),
     path("students", teacher_view.student_list, name="students"),
+    path("student/transfer", teacher_view.student_transfer, name="student_transfer"),
```

Comment:
Is there some way to get to this link from the UI, or is it a "magic link"? :)

Reply:
Well, it's still magic but most of the users know it. Anyway, there's a big trouble in access rights so those without superuser rights basically can't transfer students.